### PR TITLE
Add describe blocks to game_stats tests + add comments to module

### PR DESCRIPTION
**Pull request by Henchworm**
Update describes in spec/ add comments to game_stat module

### CODE REVIEW

File: modules/game_stats.rb [hunk 1]
```diff
@@ -1,39 +1,46 @@
 module GameStats
+
```
Comment (jaw772) [inline modules/game_stats.rb:2]: Good job with spacing. It looks really clean!

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -73,37 +73,53 @@
 
   describe 'GameStats module methods' do
 
-      it "highest_total_score" do
+    describe 'highest_total_score' do
```
Comment (jaw772) [inline spec/stat_tracker_spec.rb:76]: Good use of spacing and describe blocks!

**Review by jaw772 — APPROVED** [review]

**Review by jacksonvaldez — APPROVED** [review]

**Review by jacksonvaldez — APPROVED** [review]

File: spec/stat_tracker_spec.rb (hunk 2 above)
Comment (tjroeder) [reply spec/stat_tracker_spec.rb:76]: The extra describe blocks really clean it up nice.

**Review by tjroeder — APPROVED** [review]
Good cleanup and comments.

**Pull request merged**